ChangeLog.md

```diff
@@ -25,6 +25,9 @@ See docs/process.md for more on how version tagging works.
   emcc itself.  This enables support for C++20 in cmake projects. (#21987)
 - The version of python required to run emscripten was bumped from 3.6 to 3.8.
   (#23417)
+- The `EM_LOG_C_STACK` flag to `emscripten_log` was deprecated and the helper
+  file on which it was based (`emscripten-source-map.min.js`) deteted.  It
+  seems this feature has been broken for at least 4 years now. (#23553)
```

Comment:
```suggestion
- The `EM_LOG_C_STACK` flag to `emscripten_log` was deprecated and the helper
  file on which it was based (`emscripten-source-map.min.js`) deleted.  This
  feature (userspace source map parsing in logs) was not ported to wasm
  source maps, so it has not worked, and we have had no requests for it. This
  has no impact on browser devtools support of source maps (which work
  fully). (#23553)
```

Reply:
Done.  With a little re-phrasing of my own.